.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem?**
+A description of what the problem is. For example: I'm frustrated when [...]
+
+**Describe the solution you'd like**
+A description of what you want to happen.
+
+**Describe alternatives you've considered**
+A description of any alternative solutions or features you've considered.
```

Comment:
note that I've disabled Issues on all repos other than the community repository for right now, so there isn't any issue support in the sub-repos. But, it doesn't hurt to have this ready for if/when we enable issues on the service controller repositories.